apache-rat-core/src/main/resources/org/apache/rat/plain-rat.xsl

```diff
@@ -112,7 +112,7 @@
    char         type
     A       Archive file
     B       Binary file
-    G       Generated file
+    I       Ignored file
```

Comment:
While having a look at this file I stumbled upon "missing-headers.xsl" in the same directory. The file does not seem to be used anymore - should we remove it as the functionality of integrating snippets of "marked" files was removed, @Claudenw ?

Reply:
Having a second look it seems to referenced from multiple site-related files ..... so probably not safe to remove as it is an example transformation/report.